nx-X11/programs/Xserver/hw/nxagent/Atoms.c

```diff
@@ -95,6 +95,10 @@ static char *nxagentAtomNames[NXAGENT_NUMBER_OF_ATOMS + 1] =
 
 static XErrorHandler previousErrorHandler = NULL;
 
+const char * validateString(const char *str) {
+  return str ? str : "(null)";
+}
+
```

Comment:
As earlier discussed, I second @Ionic's suggestion to put this into Utils.h as a static inline function. The validateString macro is also used for non-Atom strings.